leetcode/intervals.md

```diff
@@ -1,19 +1,25 @@
 # Intervals

```

Comment:
Your Pull Request title: Update intervals.md

Check result:

* Pull Request title must start with prefix in ['GENERATOR', 'LEETCODE']
* Pull Request title must contain group number in ['1021', '1022', '1013', '2021', '2022']
* Pull Request title action must start with ['Added', 'Deleted', 'Refactored', 'Moved', 'Fixed']